node.js_mongodb/app.js

```diff
@@ -1,33 +1,36 @@
 const DBHelper=require('./DB_helper');
 RunProgram();
 
+var database_name="rsquared_zilla_upazilla_db";
 async function RunProgram(){
     //bulkUpload();
```

Comment:
if function is not going to be used, remove the commented out line.

Reply:
done.